core/modules/wconfig/WorkerConfig.cc

```diff
@@ -75,6 +75,7 @@ WorkerConfig::WorkerConfig(const util::ConfigStore& configStore)
       _maxTasksBootedPerUserQuery(configStore.getInt("scheduler.maxtasksbootedperuserquery", 5)),
       _maxSqlConnections(configStore.getInt("sqlconnections.maxsqlconn", 800)),
       _ReservedInteractiveSqlConnections(configStore.getInt("sqlconnections.reservedinteractivesqlconn", 50)),
+      _bufferMaxTotalGB(configStore.getInt("transmit.buffermaxtotalgb", 41)),
```

Comment:
Just curious, why `41`? Is this just a random number?

Reply:
I'm not sure what the best number is. The value of 10 in the configuration file is also a guess (backed by Andy H.), but at least it seems to work. More experimentation is needed to find the optimal value, but it isn't essential to do this immediately.